topics/statistics/tutorials/fine_tuning_protTrans/tutorial.md

```diff
@@ -0,0 +1,121 @@
+
+
+### Dephosphorylation
+Dephosphorylation is a biochemical process (post-translational modification) involving removing a phosphate group from an organic compound, typically mediated by phosphatase enzymes. This process regulates cellular functions, including signal transduction, metabolism, and protein activity. By removing phosphate groups from proteins, phosphatases counterbalance the actions of kinases, which add phosphate groups, thus maintaining the dynamic equilibrium of phosphorylation states within the cell. Dephosphorylation can activate or deactivate enzymes and receptors, alter protein-protein interactions, and influence the cellular localisation of proteins. This regulation is essential for many physiological processes, such as cell growth, differentiation, and apoptosis. Disruptions in dephosphorylation mechanisms are associated with numerous diseases, including cancer, diabetes, and neurodegenerative disorders, highlighting the importance of precise control over this process for maintaining cellular health and function. Labelled datasets specifying whether a protein sequence contains dephosphorylation sites are scarce. [Chaudhari et al.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8264445/) has used deep learning techniques to classify dephosphorylation sites but on a small dataset consisting of around 1,000 protein sequences having dephosphorylation sites at Serine (S), Threonine (T) and Tyrosine (Y). We explore fine-tuning a pre-trained ProtT5 model on a dephosphorylation dataset to showcase an approach for classifying protein sequences. In the tutorial, only those sequences having Y site as dephosphorylated are used for fine-tuning to restrict the training time to a reasonable limit. In the following sections, the GPU-enabled JupyterLab tool in Galaxy is used to fine-tune the ProtT5 model to learn and predict dephosphorylation sites.
+
+
+## JupyterLab in Galaxy Europe
+
+
+### Open JupyterLab
+
+> <hands-on-title>GPU-enabled Interactive Jupyter Notebook for Machine Learning</hands-on-title>
+>
+> - {% tool [GPU-enabled Interactive Jupyter Notebook for Machine Learning](interactive_tool_ml_jupyter_notebook) %}
+>    - *"Do you already have a notebook?"*: `Start with default notebooks`
+>    - Click *"Run Tool"*
+>
+>    > <comment-title></comment-title>
+>    >  If you do not have access to this resource in Galaxy Europe, please apply for it at: [Access GPU-JupyterLab](http://usegalaxy.eu/gpu-request). It may take a day or two to receive access.
+>    >
+>    {: .comment}
+{: .hands_on}
+
+
+### Fetch notebook and protein sequences
+
+> <hands-on-title>Fetch data from Zenodo</hands-on-title>
+>
+> 1. Create a new folder named `fine-tuning` alongside other folders such as "data", "outputs", "elyra" or you can use your favourite folder name.
```

Comment:
@anuprulez what do you think about extending the Juypter IT to take a git URL. Then when the notebooks start we clone the repo.

We could even trigger an installation of requirements.yml ... similar to binder if we like.

Reply:
@bgruening 
> what do you think about extending the Juypter IT to take a git URL. Then when the notebooks start we clone the repo.

> We could even trigger an installation of requirements.yml ... similar to binder if we like.

These are nice ideas!! Thanks!

But I need to think about how to take Git URL and requirements.yml

Reply:
> The notebooks are also welcome to live in the GTN next to the tutorial, if you do not want to risk them getting out of sync.

thanks for the idea. Where should we keep it? Creating a new folder named `notebooks` along side `images` or `workflows` folders? The notebook requires two FASTA files as well. Can we keep them along side the notebook?